mappings/net/minecraft/world/World.mapping

```diff
@@ -117,6 +117,7 @@ CLASS bbp net/minecraft/world/World
 		ARG 1 player
 		ARG 2 eventId
 		ARG 3 pos
+	METHOD a playSound (Larb;Lagv;Lxm;Lxo;FF)V
```

Comment:
Possibly should be `playSoundFromEntity`? I think just to make it clear it's different from the "at position" variant.

Reply:
:heavy_check_mark: